python-polars/README.md

```diff
@@ -0,0 +1,3 @@
+# Pandas-Groupby
```

Comment:
@hfhoffman1144 Is there a reason that Pandas-Groupby is the title here? I would suggest changing it to match the tutorial's title.

Reply:
Sorry about that, I copied the README template from another article and forgot to change the main header 😅